src/main/java/by/exadel/internship/pojo/JwtResponse.java

```diff
@@ -0,0 +1,28 @@
+package by.exadel.internship.pojo;
+
+import by.exadel.internship.dto.enums.UserRole;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.UUID;
+
+@Data
+@NoArgsConstructor
+public class JwtResponse {
+    private String token;
+    private String type = "Bearer";
```

Comment:
You decided to remove it, didn't you?

Reply:
Yes, we decided to remove type, login, email. And return only token and id